code/src/components/StyledComponents/EmptyStateStyling.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
```

Comment:
Really awesome that you made an empty state styling! Looks great and works really well!